### PR TITLE
Updated spanish localization

### PR DESCRIPTION
**Pull request by ivnc**
Updated spanish localization with two untranslated strings.
As soon as I have the galician translation I'll make the appropriate pull request.

Any problem please comment.

### CODE REVIEW

**Comment by albzan** [comment]
Hi,
Thank you.
Could you rephrase the 
"Complemento para el editor matemático Lambda"
In a way so that Lambda is the first word? In this way you can reach the addon from the list in the NVDA UI using the "L" key.
For instance, in Italian is:
"Lambda per NVDA".
What do you think?
Tnx
Alberto

**Comment by ivnc** [comment]
Done. I made also some minor changes to documentation to delete some
strings no more present at the English's.

Regards.

**Comment by ivnc** [comment]
sorry, I committed the galician translation, still in progress, to the wrong branch. I've already deleted the files and committed to the correct branch.

Regards.
